fvm/transactionVerifier.go

```diff
@@ -71,16 +75,7 @@ func (v *TransactionVerifier) verifyTransaction(
 		return err
 	}
 
-	// TODO(Janez): move disabling limits out of the verifier. Verifier should not be metered anyway.
-	// TODO(Janez): verification is part of inclusion fees, not execution fees.
-
-	// check accounts uses the state, but if the limits are too low, this might fail.
-	// we shouldn't fail here if the limits are too low as fee deduction won't happen
-	txnState.RunWithAllLimitsDisabled(
-		func() {
-			err = v.checkAccountsAreNotFrozen(tx, accounts)
-		},
-	)
+	err = v.checkAccountsAreNotFrozen(tx, accounts)
```

Comment:
why do we start to meter account status now?

Reply:
this got moved to line 40.  the entire verifyTransaction is now unlimited, rather than just this one line